Wrapping/Generators/Python/itk/support/xarray.py

```diff
@@ -34,7 +34,7 @@
 ]
 
 
-def xarray_from_image(l_image: "itkt.ImageOrImageSource") -> "xr.DataArray":
+def xarray_from_image(l_image: "itkt.ImageOrImageSource", in_place:bool=False) -> "xr.DataArray":
```

Comment:
A more conventional name would be `view` here

Reply:
Fixed